content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -363,6 +367,73 @@ will be terminated once the job backoff limit has been reached. This can make de
 from failed Jobs is not lost inadvertently.
 {{< /note >}}
 
+### Backoff limit per index {#backoff-limit-per-index}
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only configure the backoff limit per index for a Job, if you
+have the `JobBackoffLimitPerIndex` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster. A per-index backoff limit is only available for Indexed Jobs that
+use the `restartPolicy: Never`.
+{{< /note >}}
```

Comment:
Rephrased. However, I prefer not to mix the preconditions about the cluster with the requirement for `restartPolicy: Never`. Because the `restartPolicy: Never` is not a precondition, but something that the user decides to use or not. In particular, some Indexed Jobs may use it, while other don't. 

Thus, I delete this remark entirely from the note, as the note was mixing two different things. Still, the requirement is mentioned as a comment in the sample code, next to the field, so this should be enough. It is also mentioned in the API docs for the `BackoffLimitPerIndex` field, so I think this is enough for users interested in using this field. wdyt?

Reply:
I like this approach. Thanks!

Reply:
Great, resolving.